Dockerfile

```diff
@@ -0,0 +1,45 @@
+# Start from a Debian image with the latest version of Go installed
+# and a workspace (GOPATH) configured at /go.
+FROM marcbachmann/libvips:latest
```

Comment:
I've a mix feeling on depending on a random image and pulling from latest. 
It's good for testing now but let's add a TODO to change it